app/views/users/registrations/edit.html.erb

```diff
@@ -1,27 +1,29 @@
+<div class="edit-settings row">
 <h2>Edit Account Settings</h2>
+  <div class="small-12 small-centered medium-10 large-10 column">
+    <div class="change-password-form row">
+      <%= form_for(resource, as: resource_name, url: user_registration_path(resource_name), html: { method: :put }) do |f| %>
+        <%= devise_error_messages! %>
 
-<%= form_for(resource, as: resource_name, url: user_registration_path(resource_name), html: { method: :put }) do |f| %>
-  <%= devise_error_messages! %>
+        <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
+          <div>Currently waiting confirmation for: <%= resource.unconfirmed_email %></div>
+        <% end %>
 
-  <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
-  <div>Currently waiting confirmation for: <%= resource.unconfirmed_email %></div>
-  <% end %>
+            <h3>Change my password</h3>
 
-  <h3>Change my password</h3>
+            <div><%= f.label :password %>
+                <%= f.password_field :password, autocomplete: "off" %></div>
 
-  <div><%= f.label :password %> <i>
-      <%= f.password_field :password, autocomplete: "off" %></div>
+            <div><%= f.label :password_confirmation %>
+                <%= f.password_field :password_confirmation, autocomplete: "off" %></div>
```

Comment:
placeholder: "Re-type your new password just to make sure it's 100% correct" or something similar